packages/braintree-integration/src/braintree-fastlane/braintree-fastlane-utils.ts

```diff
@@ -341,4 +344,19 @@ export default class BraintreeFastlaneUtils {
             await this.paymentIntegrationService.selectShippingOption(selectedOption.id);
         }
     }
+
+    private async updateShippingAddressHandler(
+        cart: Cart,
+        shippingAddresses: CustomerAddress[],
+        shouldSetShippingOption: boolean,
+        isFastlaneShippingOptionAutoSelectEnabled?: boolean,
+    ): Promise<void> {
+        if (shippingAddresses.length > 0 && cart.lineItems.physicalItems.length > 0) {
```

Comment:
Please, do not use such conditions on method level, because it is not quite transparent, that this method will do nothing when cart does not have any physical item